### ISSUE TITLE
Custom content

### ISSUE THREAD

**Issue by tumidi** [innodoc/innodoc-webapp#59]
There is content that is integrated in different locations in the web app. E.g. an optional **About page** should show up automatically in the navigation and/or footer but not in the table of content. Other examples include:

#### Examples

- About page
- Copyright/license
- Usage instructions
- Imprint, collaborators, authors
- Contact page
- Disclaimer

These are mostly custom pages that should be accessible via dynamic Next.js routes. Apart from that there is also content that is inserted at certain places (e.g. license or contributors in footer).

###### JSON (compiled source)

```json
  "custom_content": [
    {
      "name": "about",
      "title_de": "Über den Kurs",
      "title_en": "About the course",
      "link_in_nav": true,
      "link_in_footer": true
    },
    {
      "name": "license",
      "title_de": "Lizenz",
      "title_en": "License",
      "place_in_footer": true
    }
  ]
```

##### See also

- #32
- innodoc/innoconv#31
- innodoc/innoconv#32

**Comment by tumidi** [comment]
mentioned in issue innoconv#31

**Comment by tumidi** [comment]
mentioned in issue innoconv#32

**Comment by tumidi** [comment]
In GitLab by @ghost on Dec 4, 2018, 12:02

created branch [`44-custom-content`](https://gitlab.tubit.tu-berlin.de/innodoc/innodoc-webapp/compare/master...44-custom-content)

 *By Dennis Otrombowsky on 2018-12-04T11:02:55 (imported from GitLab)*

**Comment by tumidi** [comment]
mentioned in commit 4ea05d685349c4f15702d7b717d7ba12060e34dc

**Comment by tumidi** [comment]
mentioned in commit 8f8c0c9a37fb172f399d2db9451e5c4bfeb87d0a

**Comment by tumidi** [comment]
closed via commit 8f8c0c9a37fb172f399d2db9451e5c4bfeb87d0a
